doc/languages.rst

```diff
@@ -10,103 +10,167 @@ Programming languages
 
 * ActionScript
 * Ada
+* Agda (incl. literate)
+* `Alloy <http://alloy.mit.edu>`_
+* `AMPL <http://ampl.com>`_
 * ANTLR
+* APL
 * AppleScript
 * Assembly (various)
 * Asymptote
 * `Augeas <http://augeas.net>`_
+* AutoIt
 * Awk
 * BBC Basic
 * Befunge
+* BlitzBasic
 * `Boa <http://boa.cs.iastate.edu/docs/index.php>`_
 * Boo
+* `Boogie <https://boogie.codeplex.com/>`_
 * BrainFuck
-* C, C++
+* C, C++ (incl. dialects like Arduino)
 * C#
+* `Chapel <http://chapel.cray.com>`_
 * `Charm++ CI <http://charmplusplus.org/>`_
+* Cirru
+* Clay
+* `Clean <https://clean.cs.ru.nl/Clean>`_
 * Clojure
 * CoffeeScript
 * ColdFusion
 * Common Lisp
-* Coq
+* Component Pascal
+* `Coq <https://coq.inria.fr/>`_
+* Croc (MiniD)
 * Cryptol (incl. Literate Cryptol)
-* `Crystal <http://crystal-lang.org>`_
-* `Cython <http://cython.org>`_
-* `D <http://dlang.org>`_
+* `Crystal <https://crystal-lang.org>`_
+* Cypher
+* `Cython <https://cython.org>`_
+* `D <https://dlang.org>`_
 * Dart
 * DCPU-16
 * Delphi
-* Dylan
-* `Elm <http://elm-lang.org/>`_
+* Dylan (incl. console)
+* Eiffel
+* `Elm <https://elm-lang.org/>`_
+* Emacs Lisp
 * Email
-* Erlang
-* `Ezhil <http://ezhillang.org>`_ Ezhil - A Tamil programming language
+* Erlang (incl. shell sessions)
+* `Ezhil <http://ezhillang.org>`_
 * Factor
 * Fancy
+* Fantom
 * `Fennel <https://fennel-lang.org/>`_
 * `FloScript <http://ioflo.com/>`_
 * Fortran
 * `FreeFEM++ <https://freefem.org/>`_
 * F#
 * GAP
 * Gherkin (Cucumber)
-* GL shaders
+* GLSL shaders
+* `Golo <http://golo-lang.org/>`_
+* Gosu
 * Groovy
 * `Haskell <http://www.haskell.org>`_ (incl. Literate Haskell)
 * HLSL
 * `HSpec <http://hackage.haskell.org/package/hspec>`_
+* Hy
 * IDL
+* Idris (incl. Literate Idris)
+* Igor Pro
 * Io
+* Jags
 * Java
 * JavaScript
-* Lasso
-* LLVM
+* Jasmin
+* Jcl
+* `Julia <https://julialang.org>`_
+* Kotlin
+* Lasso (incl. templating)
+* Limbo
+* LiveScript
 * Logtalk
-* `Lua <http://www.lua.org>`_
+* Logos
+* `Lua <https://lua.org>`_
+* Mathematica
 * Matlab
-* MiniD
 * Modelica
 * Modula-2
+* Monkey
+* `Monte <https://monte.readthedocs.io/>`_
+* MoonScript
 * MuPad
+* NASM
 * Nemerle
+* NesC
+* NewLISP
 * Nimrod
+* `Nit <http://nitlanguage.org>`_
 * Notmuch
+* NuSMV
 * Objective-C
 * Objective-J
 * Octave
 * OCaml
+* Opa
+* OpenCOBOL
+* `ParaSail <http://www.parasail-lang.org>`_
+* Pawn
 * PHP
-* `Perl 5 <http://perl.org>`_ and `Perl 6 <https://perl6.org>`_
+* `Perl 5 <https://perl.org>`_ and `Perl 6 <https://perl6.org>`_
```

Comment:
Perl6 is Raku now? Not sure if we care yet: https://raku.org/

Reply:
yeah, will fix